python/ommx/src/instance.rs

```diff
@@ -30,7 +30,7 @@ impl Instance {
     }
     
     pub fn constraint_hints(&self) -> ConstraintHints {
```

Comment:
Do not return ConstraintHints directly. Rather, enclose it inside `Bound<'_, >` to avoid cloning.

```suggestion
    pub fn constraint_hints(&self) -> Bound<'_, ConstraintHints> {
```